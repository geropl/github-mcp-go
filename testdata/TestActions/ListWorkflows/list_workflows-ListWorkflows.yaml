---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 0
        transfer_encoding: []
        trailer: {}
        host: api.github.com
        remote_addr: ""
        request_uri: ""
        body: ""
        form: {}
        headers:
            Accept:
                - application/vnd.github.v3+json
            User-Agent:
                - go-github/v69.2.0
            X-Github-Api-Version:
                - "2022-11-28"
        url: https://api.github.com/repos/geropl/github-mcp-go-test/actions/workflows?page=1&per_page=30
        method: GET
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: '{"total_count":1,"workflows":[{"id":149593961,"node_id":"W_kwDOOEmhcs4I6p9p","name":"Test Data Generation Workflow","path":".github/workflows/test-workflow.yml","state":"active","created_at":"2025-03-13T17:38:47.000+01:00","updated_at":"2025-03-13T17:38:47.000+01:00","url":"https://api.github.com/repos/geropl/github-mcp-go-test/actions/workflows/149593961","html_url":"https://github.com/geropl/github-mcp-go-test/blob/main/.github/workflows/test-workflow.yml","badge_url":"https://github.com/geropl/github-mcp-go-test/workflows/Test%20Data%20Generation%20Workflow/badge.svg"}]}'
        headers:
            Access-Control-Allow-Origin:
                - '*'
            Access-Control-Expose-Headers:
                - ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
            Cache-Control:
                - private, max-age=60, s-maxage=60
            Content-Security-Policy:
                - default-src 'none'
            Content-Type:
                - application/json; charset=utf-8
            Date:
                - Thu, 13 Mar 2025 16:43:54 GMT
            Etag:
                - W/"14022cde3eaf5856245b9078fc926a48814390663ae8408a46c72c1949b74d1b"
            Github-Authentication-Token-Expiration:
                - 2025-04-12 17:32:16 +0200
            Referrer-Policy:
                - origin-when-cross-origin, strict-origin-when-cross-origin
            Server:
                - github.com
            Strict-Transport-Security:
                - max-age=31536000; includeSubdomains; preload
            Vary:
                - Accept, Authorization, Cookie, X-GitHub-OTP,Accept-Encoding, Accept, X-Requested-With
            X-Accepted-Github-Permissions:
                - actions=read
            X-Content-Type-Options:
                - nosniff
            X-Frame-Options:
                - deny
            X-Github-Api-Version-Selected:
                - "2022-11-28"
            X-Github-Media-Type:
                - github.v3; format=json
            X-Github-Request-Id:
                - 579B:33CFBA:449222C:46B8E7B:67D30B4A
            X-Ratelimit-Limit:
                - "5000"
            X-Ratelimit-Remaining:
                - "4870"
            X-Ratelimit-Reset:
                - "1741885609"
            X-Ratelimit-Resource:
                - core
            X-Ratelimit-Used:
                - "130"
            X-Xss-Protection:
                - "0"
        status: 200 OK
        code: 200
        duration: 199.250981ms
