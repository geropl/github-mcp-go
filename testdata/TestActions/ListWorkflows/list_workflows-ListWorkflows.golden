{
  "output": "# Workflows\n\nFound 1 workflows.\n\n## 1. Test Data Generation Workflow\n\n**ID:** 149593961  \n**Path:** .github/workflows/test-workflow.yml  \n**State:** active  \n**Created:** Thu, 13 Mar 2025 17:38:47 +0100  \n**Updated:** Thu, 13 Mar 2025 17:38:47 +0100  \n**URL:** https://github.com/geropl/github-mcp-go-test/blob/main/.github/workflows/test-workflow.yml  \n\n",
  "err": ""
}